imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -65,24 +71,27 @@ class CaseExplorer extends Component {
       })
     }
   }
+
   render () {
     const { isLoading, dispatch, match } = this.props
-    const { unitsDict, showOpen, expandedUnits } = this.state
+    const { unitsDict, showOpen, expandedUnits, assignedToMe } = this.state
     const casesFilterFunc = showOpen ? x => !isClosed(x) : x => isClosed(x)
-
     return (
       <div className='flex flex-column roboto overflow-hidden flex-grow h-100'>
         <div className='bb b--black-10 overflow-auto flex-grow'>
           <div className='flex pl3 pv3 bb b--very-light-gray'>
             <div onClick={() => this.handleStatusClicked(true)} className={'f6 fw5 ' + (showOpen ? 'mid-gray' : 'silver')}> Open </div>
             <div onClick={() => this.handleStatusClicked(false)} className={'f6 fw5 ml4 ' + (!showOpen ? 'mid-gray' : 'silver')}> Closed </div>
+            <div onClick={() => this.handleAssignedClicked()} className={'f6 fw5 ml4 ' + (assignedToMe ? 'mid-gray' : 'silver')}> Assigned To Me</div>
           </div>
           {!isLoading && Object.keys(unitsDict)
             .reduce((all, unitTitle) => {
               const isExpanded = expandedUnits.includes(unitTitle)
               const allCases = unitsDict[unitTitle]
-              const hasCases = allCases.filter(casesFilterFunc)
-              if (hasCases.length > 0) {
+              const myCaseFilter = allCases.filter(caseItem => caseItem.assignee === assignedToMe)
+              const myCases = assignedToMe ? myCaseFilter : allCases
+              const openCaseFilter = myCases.filter(casesFilterFunc)
```

Comment:
It will be much neater if you could combine both filter funcs so you'll only need to use `.filter(...)` once

Reply:
Same comment about naming... something like `openCases` is a better name

Reply:
`openCaseFilter` displays either open or closed cases. I changed name from`hasCases` to `openCaseFilter` to pinpoint that distinction. What name would you suggest?